client/src/components/Details/Details.css

```diff
@@ -0,0 +1,3 @@
+button {
+	background-color: red;
```

Comment:
Please use a specific class name when using CSS. If not this style will affect all button in the project.